### ISSUE TITLE
Define @@ in template-for-success-criterion.md

### ISSUE THREAD

**Issue by shawnthompson** [w3c/wcag2ict#59]
Could we define what is meant when we use `@@` in the [template-for-success-criterion.md](https://github.com/w3c/wcag2ict/blob/main/.github/ISSUE_TEMPLATE/template-for-success-criterion.md).

Example:

>From [Success Criterion @@ x.x.x @@](@@ URL for the SC in WCAG 2.2 @@):

**Comment by bruce-usab** [comment]
From previous AGWG work, I understand @@ to mean "look here".   The Word version of the template has comment bubbles, so that is helpful and instructive, but (of course) we need a different convention for Markdown templates.  And I think I heard Mary Jo say that the Word version of the template needs some updating?

@maryjom &mdash; where did the SC you wrote up during the call end up?  I cannot find it in the repo.

In addition to a blank template, I know I always appreciated a completed doc to copy/paste from.

**Comment by maryjom** [comment]
@shawnthompson, @bruce-usab is correct that @@ is the typical W3C notation for "something needs updating here". Content isn't considered complete until all @'s are addressed in the document.  I have added an explanation in the [GitHub wiki version of the template](https://github.com/w3c/wcag2ict/wiki/Template-for-Success-Criterion-using-wiki-markdown) of what it means and what to do with an example of a line that had a bunch of @ signs and they were replaced with the appropriate text. If that explanation is clear, I'll update the issue template with those same instructions.

**Comment by maryjom** [comment]
Closing. I think this issue has been addressed.
